diagnostic.md

```diff
@@ -5,20 +5,25 @@ Record your responses inside the fenced code blocks below each question.
 1.  Give an example of a visual hierarchy that could be modeled with components.
 
     ```md
-    <!-- your response here -->
+    For data down the information of the model goes to the component and then it is sent to the template.
+    Say you have information on daily chores and you want to display the information as a list.  You can use the information from the route and pass it to a component to render a list of chores.
+    But when it bubbles up the information stays in component unless you send the action out to the route.
     ```
 
 1.  What is the command to generate a new component called '`my-map`'?
 
     ```sh
-    # your response here
+    ember g component my-map
     ```
 
 1.  What files are edited to produce a component, and what are their
     responsibilities?
 
     ```md
-    <!-- your response here -->
+    You would want to edit the components files (component.js and the template.hbs) and in the template of whatever path you want to pass in the component and bind it to data in handlebars.
+    {{component data=data}}
+
+    Component file responsibilities: component file allows you to pass html bindings, event actions, or click handlers to the template.  The template just renders the information in handlebars.
```

Comment:
Yes.  The component does have a lot of responsibility, but the template is important as well.  The `template.hbs` is the one who is responsible for binding data, and binding and dispatching actions.  The `component.js` is responsible for defining actions and attributes.